### ISSUE TITLE
Update pep8

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#91]
In GitLab by @rfleschenberg on Mar 25, 2015, 07:10

The pep8 tool that comes bundled with flake8 is outdated and reports false warnings. You should update.

E.g.:
```
# foo.py
a = 1
b = 2
print a*b + b*b

flake8 foo.py
foo.py:4:8: E225 missing whitespace around operator
foo.py:4:14: E225 missing whitespace around operator
```

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Mar 25, 2015, 09:50

Please check existing (closed and open) issues before filing new ones. We're specifically pinning to <1.6.0 because it breaks other features. If you want to use newer (broken) versions of pep8, pin to flake8 2.3.0

**Comment by asottile** [comment]
In GitLab by @rfleschenberg on Mar 26, 2015, 02:54

I searched the existing issues, but could not find a related one. Can you point me to it?

So, you're "specifically pinning to" what? A certain pep8 version, I suppose? I realize that. The point is that the pep8 version you are using is buggy. It contradicts PEP 8.
See the output I gave above vs. https://www.python.org/dev/peps/pep-0008/#other-recommendations

You should either update your pep8 version or backport the fix for this bug.

**Comment by asottile** [comment]
In GitLab by @rfleschenberg on Mar 26, 2015, 02:59

OK. Now I read a little more on the other issues and realize how they are related. Sorry. I was also thinking you had actually *bundled* pep8, but it seems I was looking at the wrong source? Anyway, closing. I hope this gets fixed soon.

**Comment by rfleschenberg** [comment]
@asottile Thanks! @sigmavirus24, I want to apologize for my comments. Reading them again now after 6 years have passed, I realize I sounded like an entitled asshole :( Sorry.

**Comment by sigmavirus24** [comment]
Appreciate it @rfleschenberg

We all have bad moments. I've definitely had mine and I'm trying to avoid having more by trying to figure out my stress coping strategies right now.
